src/integrators/controllers.jl

```diff
@@ -166,7 +171,7 @@ end
 function step_reject_controller!(integrator,alg::QNDF)
   #append no. of consecutive failed steps
   integrator.cache.consfailcnt += 1
-  integrator.dt = integrator.dt/integrator.qold
+  integrator.dt = integrator.qold
```

Comment:
Why is qold used as dt? I think qold is unitless

Reply:
I have changed some definitions here https://github.com/SciML/OrdinaryDiffEq.jl/pull/1173/files#diff-c90c88b38a0b685e7edd6abc54fa1ef8R147-R151

Reply:
You would probably need to change definitions at many places for q and qold. Also this wont work with units. qold and q are used as ratios, make sure that nothing funny is happening. Also this would make a problem for composite algorithms, but that can be handled later

Reply:
https://github.com/SciML/OrdinaryDiffEq.jl/blob/master/src/integrators/controllers.jl#L33
https://github.com/SciML/OrdinaryDiffEq.jl/blob/master/src/integrators/controllers.jl#L108
I tried to follow what was done in the standard step-size controller and how q and old were used by `step_accept_controller` and `step_reject_controller`. Let me know if it's okay.

Reply:
ohh, that's... strange. I think we have a separate issue then. Surprised this works with units.

Reply:
Yup there's a difference of `q` & `qold` as to how algorithms use them.

Reply:
yeah... it's kind of kludgy and probably is worth refactoring.

Reply:
I would definitely like to look into it!